whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/NrtmAclLimitHandler.java

```diff
@@ -1,14 +1,9 @@
 package net.ripe.db.whois.nrtm;
 
+import io.netty.channel.*;
```

Comment:
Use explicit imports rather than '*' as it's clearer to see what changed in a diff

Reply:
updated